client/templates/testSmartContractTemplate.ejs

```diff
@@ -1,17 +1,77 @@
+/*
+* Use this file for functional testing of your smart contract. 
+* Fill out the arguments and return values for a function and 
+* use a test runner to run these tests locally. 
+* All transactions defined in your smart contract are used here 
+* to generate tests, including those functions that would 
+* normally only be used on instantiate and upgrade operations.
+* This basic test file can also be used as the basis for building 
+* further functional tests to run as part of a continuous 
+* integration pipeline, or for debugging locally deployed smart 
+* contracts by invoking/submitting individual transactions. 
+*/
+
 const assert = require('assert');
+const uuid = require('uuid/v4');
+const fabricNetwork = require('fabric-network');
+const yaml = require('js-yaml');
+const fs = require('fs-extra');
 
 describe('<%=chaincodeLabel%>' , () => {
 
-    before(() => {
+    const gateway = new fabricNetwork.Gateway();
+    let connectionProfile;
+    const wallet = new fabricNetwork.InMemoryWallet();
+    const identityName = uuid();
```

Comment:
i don't think you need this as a uuid, you could just use the name of the smart contract.